### PR TITLE
Added template and script to check local files

### PR DESCRIPTION
**Pull request by donaldzou**
# How to use

`python3 verify_locale_files.py ` in `src/static/locale`

## Example

```
(venv) donaldzou@Mac locale % python3 verify_locale_files.py 
========================================================
| WGDashboard Locale File Verification [by @donaldzou] |
========================================================

Active Languages
        [Language #1] Czech
        [Language #2] German
        [Language #3] English
        [Language #4] Italian
        [Language #5] Dutch
        [Language #6] Russian
        [Language #7] Turkish
        [Language #8] Ukrainian
        [Language #9] Chinese (Simplified)
        [Language #10] Chinese (Traditional)
Select the language number you want to verify or enter [q] to exit: 1

Selected Language: Czech

        [Missing Translations] Simply copy the JSON below into the locale file and translate them in :)
                "Signing In\.\.\.": "",
                "Multi-Factor Authentication \(MFA\)": "",
                "Are you sure to delete this API key\?": "",
                "When should this API Key expire\?": "",
                "Creating\.\.\.": "",
                "Turning On\.\.\.": "",
                "Turning Off\.\.\.": "",
                "Search Peers\.\.\.": "",
                "\(Required for QR Code and Download\)": "",
                "\(Required\)": "",
                "This peer does not have any job yet\.": "",
                "No active job at the moment\.": "",
                "Sharing\.\.\.": "",
                "Stop Sharing\.\.\.": "",
                "Restricting\.\.\.": "",
                "Allowing Access\.\.\.": "",
                "Download \& QR Code is not available due to no private key set for this peer": "",
                "By adding peers by bulk, each peer's name will be auto generated, and Allowed IP will be assign to the next available IP\.": "",
                "How many peers you want to add\?": "",
                "Adding\.\.\.": "",
                "Dashboard IP Address \& Listen Port": ""

        [Deprecated Translations] Simply remove the translation below :)1
                "Signing In...": "Přihlašování..."
                "Multi-Factor Authentication (MFA)": "Vícefaktorové ověření (MFA)"
                "Are you sure to delete this API key?": "Určitě chcete smazat tento API klíč?"
                "When should this API Key expire?": "Kdy má tento API klíč vypršet?"
                "Creating...": "Tvořím..."
                "Turning On...": "Zapínám..."
                "Turning Off...": "Vypínám..."
                "Search Peers...": "Hledat peers..."
                "(Required for QR Code and Download)": "(Nutné pro QR kód a stažení)"
                "(Required)": "(Povinné)"
                "This peer does not have any job yet.": "Tento peer ještě nemá žádný úkol."
                "No active job at the moment.": "Žádný úkol v současnosti neprobíhá."
                "Sharing...": "Sdílím..."
                "Stop Sharing...": "Ukončit sdílení..."
                "Restricting...": "Omezuji..."
                "Allowing Access...": "Povoluji přístup..."
                "Download & QR Code is not available due to no private key set for this peer": "Stažení a QR kód nejsou k dispozici, peer nemá nastavený soukromý klíč."
                "By adding peers by bulk, each peer's name will be auto generated, and Allowed IP will be assign to the next available IP.": "Při hromadném importu bude jméno každého peera vygenerované automaticky, a povolená IP adresa bude nastavena na další v pořadí"
                "How many peers you want to add?": "Kolik peerů chcete přidat?"
                "Adding...": "Přidávám..."
```

### CODE REVIEW

**Comment by donaldzou** [comment]
Hi @DaanSelen, when u are free, do you mind going through this PR and see if there's anything to improve and streamline the localization process?

Thank u!

**Comment by DaanSelen** [comment]
I'll go through it, give me some time!

**Comment by DaanSelen** [comment]
https://github.com/donaldzou/WGDashboard/pull/492

**Review by DaanSelen — CHANGES_REQUESTED** [review]
Couple points:
I don't think the PL is alphabetical behind the ZH-HK?
The indentation in `pl-pl.json` is not consistent. (content looks good)
The Dutch translation changed `...` to `\\.\\.\\.`

File: src/static/locale/nl-nl.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "Password": "Wachtwoord",
   "OTP from your authenticator": "OTP van uw authenticator",
   "Sign In": "Inloggen",
-  "Signing In...": "Inloggen...",
+  "Signing In\\.\\.\\.": "Inloggen...",
```
Comment (DaanSelen) [inline src/static/locale/nl-nl.json:7]: Which one is leading?

File: src/static/locale/pl.json [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+	"Select a backup you want to restore": "Wybierz kopię zapasową którą chcesz przywrócić",
+	"Click to change a backup": "Wybierz kopię zapasową",
+	"Selected Backup": "Wybrana kopia zapasowa",
+	"You don't have any configuration to restore": "Nie posiadasz żadnej konfiguracji do przywrócenia",
+	"Help": "Pomoc",
+	"Backup": "Kopia Zapasowa",
+	"([0-9].*) Backups?": "$1 Kopii Zapasowych?",
+	"Yes": "Tak",
+	"No": "Nie",
+    "Backup not selected": "Nie wybrano kopii zapasowej",
```
Comment (DaanSelen) [inline src/static/locale/pl.json:258]: Indentation looks to be forgotten.

File: src/static/locale/uk.json [hunk 3]
```diff
@@ -236,6 +236,7 @@
 	"IP Address": "IP-адреса",
 	"Enter IP Address / Hostname": "Введіть IP-адресу / ім’я хоста",
 	"IP Address / Hostname": "IP-адресу / ім’я хоста",
+	"Dashboard IP Address \\& Listen Port": "",
```
Comment (DaanSelen) [inline src/static/locale/uk.json:239]: Empty line in Ukranian.

**Comment by donaldzou** [comment]
Yes I found out a bug in the python script that it escaped 2 back slashes when printed out. Need to figure that out :)

**Comment by DaanSelen** [comment]
> Yes I found out a bug in the python script that it escaped 2 back slashes when printed out. Need to figure that out :)

Did all things get resolved?

**Comment by donaldzou** [comment]
> > Yes I found out a bug in the python script that it escaped 2 back slashes when printed out. Need to figure that out :)
> 
> Did all things get resolved?

Yes, now what's left to do is fixing the empty translations that got replaced by the regex escaped keys:

"Sharing..." ==> "Sharing\\.\\.\\."

**Comment by donaldzou** [comment]
The current script will insert the new translation and remove deprecated ones directly from the file.

**Review by DaanSelen — APPROVED** [review]

**Pull request merged**